types/bitcoinjs-lib/index.d.ts

```diff
@@ -1,114 +1,234 @@
-// Type definitions for bitcoinjs-lib 2.3
```

Comment:
Also, this is a significant change. Please consider adding a _tslint.json_ containing `{ "extends": "dtslint/dt.json" }` to catch linter errors.